lib/api/helpers.js

```diff
@@ -4,12 +4,22 @@ import { FlowRouter } from "meteor/kadira:flow-router-ssr";
 import { Shops } from "/lib/collections";
 
 /* eslint no-unused-vars: 0 */
-//
-//  TODO review this slugify import in lib/api/helpers
-//
+
+// Export Reaction using commonJS style for common libraries to use Reaction
+// https://docs.meteor.com/packages/modules.html#CommonJS
+let Reaction;
```

Comment:
Yeah, I guess that makes sense.  We'd probably want to update all of the imports all over the app too.  And yes, it should probably extend with Collections/Schemas there instead of doing it redundantly on both the [client](https://github.com/reactioncommerce/reaction/blob/development/client/modules/core/index.js#L10-L11) and [server](https://github.com/reactioncommerce/reaction/blob/development/server/api/core/index.js#L14-L15).

However, I definitely don't think that's the right file for that stuff.  It should probably live in `/lib/api/core.js`.